### PR TITLE
Add signout button, update colors, remove error

### PR DESCRIPTION
**Pull request by NovaPlasm**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const MyButton = Object.assign({}, AmplifyTheme.button, { backgroundColor: '#65b
 const MyTheme = Object.assign({}, AmplifyTheme, { button: MyButton });
 
 Amplify.configure(config);
+Analytics.disable();
```
Comment (acgrobman) [inline App.js:14]: I'm taking it this is what gets rid of the weird YellowBox we were getting?

File: pages/SettingsScreen.js [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+import { Button } from 'react-native-elements';
  
-export default class Screen3 extends Component {
+export default class Settings extends Component {
   //Screen3 Component
   render() {
     return (
       <View style={styles.MainContainer}>
-        <Text style={{ fontSize: 23 }}> Screen 3 </Text>
+        <Button title="Sign out" onPress={() => Auth.signOut()} />
```
Comment (acgrobman) [inline pages/SettingsScreen.js:14]: Is there a cool icon we could add to this button?

**Review by acgrobman — APPROVED** [review]

File: App.js (hunk 1 above)
Comment (NovaPlasm) [reply App.js:14]: It got rid of the invalid promise error that we were filtering out

File: pages/SettingsScreen.js (hunk 2 above)
Comment (NovaPlasm) [reply pages/SettingsScreen.js:14]: Yea probably, i'll look

**Pull request merged**